apps/meteor/tests/end-to-end/api/05-chat.js

```diff
@@ -1129,6 +1129,44 @@ describe('[Chat]', function () {
 				})
 				.end(done);
 		});
+
+		it('should add quote attachments to a message', async () => {
+			const siteUrl = process.env.SITE_URL || process.env.TEST_API_URL || 'http://localhost:3000';
+			const quotedMsgLink = `${siteUrl}/channel/general?msg=${message._id}`;
+			request
+				.post(api('chat.update'))
+				.set(credentials)
+				.send({
+					roomId: 'GENERAL',
+					msgId: message._id,
+					text: `Testing quotes ${quotedMsgLink}`,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((res) => {
+					expect(res.body).to.have.property('success', true);
+					expect(res.body).to.have.nested.property('message.msg', `Testing quotes ${quotedMsgLink}`);
+					expect(res.body.message).to.have.property('attachments').that.is.an('array').that.has.lengthOf(1);
+				});
+		});
+
+		it('should erase old quote attachments when updating a message', async () => {
```

Comment:
Can we add a test case for when we are updating or replacing or both the existing quotes of a message?